backend/tournesol/utils/cache.py

```diff
@@ -0,0 +1,25 @@
+from django.conf import settings
+from django.views.decorators.cache import cache_page
+from django.utils import translation
```

Comment:
We can swap these two import statements to keep them alphabetically ordered as recommended by `isort` ( which is recommended by Django ).

```suggestion
from django.utils import translation
from django.views.decorators.cache import cache_page
```

see: https://docs.djangoproject.com/en/dev/internals/contributing/writing-code/coding-style/#imports

Reply:
@GresilleSiffle Do you want to add isort to run automatically in Git-Hook / CI ? 

https://pycqa.github.io/isort/docs/configuration/git_hook.html

Reply:
Yes please : )

If no objection I would like too. I'll just finish the other tasks I'm working on before.

Reply:
At first I thought that could be enforced using `pylint`, but it seems pylint only checks that standard library modules are imported before external ones.

Still, a wider issue could be about enforcing `pylint` checks, possibly only a subset of the current rules if they are too constraining. For now only a global score is computed, with a threshold at 8.5 / 10.